core-metastore/src/main/java/org/zalando/nakadi/repository/db/SubscriptionDbRepository.java

```diff
@@ -241,4 +292,235 @@ public String toString() {
                     '}';
         }
     }
+
+    private ListResult listBackwardToken(
+            final StringBuilder queryBuilder, final List<String> clauses, final List<Object> params,
+            final Token token, final int limit) {
+
+        clauses.add(" s_id " + token.getTokenType().dbClause + " ? ");
+        params.add(token.getSubscriptionId());
```

Comment:
maybe these 2 lines should be moved into the calling method (not to repeat yourself)